src/components/Todos/model.js

```diff
@@ -93,12 +100,28 @@ function makeModification$(actions) {
   );
 }
 
+// THIS IS THE MODEL FUNCTION
+// It expects the actions coming in from the todo items and
+// the initial localStorage data.
 function model(actions, sourceTodosData$) {
+  // THE BUSINESS LOGIC
+  // Actions are passed to the `makeModification$` function
+  // which creates a stream of modification functions that needs
+  // to be applied on the todoData when an action happens.
   const modification$ = makeModification$(actions);
 
+  // RETURN THE MODEL DATA
+  // Take the initial localStorage data stream...
   return sourceTodosData$
+    // ...and concatenate it with the stream of modification
+    // functions...
     .concat(modification$)
+    // ...and apply every modification function on the
+    // current todosData which is the "accumulater"
+    // of this `.scan` function.
     .scan((todosData, modFn) => modFn(todosData))
+    // Always serve any subscriber the latest state that
+    // was flowing through this Observable.
```

Comment:
`share` makes it hot, which means it multicasts/broadcasts the Observable for many observers.